app/views/contexts/section_summary_context.py

```diff
@@ -0,0 +1,160 @@
+from typing import Iterator, Mapping
+
+from flask import url_for
+
+from app.questionnaire import QuestionnaireSchema
+from .context import Context
+from .list_context import ListContext
+from .summary import Group
+
+
+class SectionSummaryContext(Context):
+    def __init__(
+        self, language, schema, answer_store, list_store, progress_store, metadata
+    ):
+        super().__init__(
+            language, schema, answer_store, list_store, progress_store, metadata
+        )
+        self.list_context = ListContext(
+            self._language,
+            self._schema,
+            self._answer_store,
+            self._list_store,
+            self._progress_store,
+            self._metadata,
+        )
+
+    def __call__(self, current_location):
```

Comment:
This whole bit just confuses me. So if our current location doesn't have a block id, like a section summary, its set as not able to collapse. However if it's a section summary why would we then build a summary for it.

Reply:
Discussed offline :+1:

Reply:
It would be good to briefly describe here the outcome of what was discussed.